packages/twilio-run/src/config/deploy.ts

```diff
@@ -74,6 +74,7 @@ export async function getConfigFromFlags(
       (externalCliOptions && externalCliOptions.accountSid) ||
       undefined,
     environmentSuffix: flags.environment,
+    region: flags.region,
```

Comment:
The calls to `getFunctionServiceSid` are missing the `region` passed in as the cache from the `twilioDeployInfo` file are not being read. Same applies in the other files.